cas-server-core/src/main/java/org/jasig/cas/CentralAuthenticationServiceImpl.java

```diff
@@ -168,6 +168,10 @@
     @NotNull
     private final UniqueTicketIdGenerator defaultServiceTicketIdGenerator
             = new DefaultUniqueTicketIdGenerator();
+
+    /** Whether we should track the most recent session by keeping the latest service ticket. */
+    private boolean onlyTrackMostRecentSession = true;

```

Comment:
As it's a new major version (4.2) and as I'm confident this change won't break anything in almost use cases, I choose to enable it by default.